gulpfile.js

```diff
@@ -1,13 +1,15 @@
-var gulp = require('gulp');
-var tslint = require('gulp-tslint');
-var exec = require('child_process').exec;
-var gulp = require('gulp-help')(gulp);
-var tsconfig = require('gulp-tsconfig-files');
-var path = require('path');
-var inject = require('gulp-inject');
-var gulpSequence = require('gulp-sequence');
-var rename = require('gulp-rename');
-var del = require('del');
+var gulp = require('gulp'),
+    tslint = require('gulp-tslint'),
+    exec = require('child_process').exec,
+    gulp = require('gulp-help')(gulp),
+    tsconfig = require('gulp-tsconfig-files'),
+    path = require('path'),
+    inject = require('gulp-inject'),
+    gulpSequence = require('gulp-sequence'),
+    rename = require('gulp-rename'),
+    del = require('del'),
+    jasmine = require('gulp-jasmine'),
+    watch = require('gulp-watch');
```

Comment:
the change of the way that I rewrite the assignations it's not related with the PR. Despite that, jasmine and watch modules (line 11 and 12 respectively) are necessary to perform the tests.